Source/HarmonyPatches/Patches_PermissionFix/Patch_ItemIsForbidden.cs

```diff
@@ -9,44 +9,45 @@
 
 namespace PrisonLabor.HarmonyPatches.Patches_PermissionFix
 {
-    /// <summary>
-    ///     Add checking if food is reserved by prisoner
-    /// </summary>
-    [HarmonyPatch(typeof(ForbidUtility))]
-    [HarmonyPatch("IsForbidden")]
-    [HarmonyPatch(new[] {typeof(Thing), typeof(Pawn)})]
-    internal class Patch_ItemIsForbidden
-    {
-        private static IEnumerable<CodeInstruction> Transpiler(ILGenerator gen, MethodBase mBase,
-            IEnumerable<CodeInstruction> instr)
-        {
-            var endOfPatch = gen.DefineLabel();
-            yield return new CodeInstruction(OpCodes.Ldarg_0);
-            yield return new CodeInstruction(OpCodes.Ldarg_1);
-            yield return new CodeInstruction(OpCodes.Call, typeof(Patch_ItemIsForbidden).GetMethod(nameof(CustomForbidConditions)));
-            yield return new CodeInstruction(OpCodes.Brfalse, endOfPatch);
-            yield return new CodeInstruction(OpCodes.Ldc_I4_1);
-            yield return new CodeInstruction(OpCodes.Ret);
+    // TODO: what does this do?!!
```

Comment:
prisoners and colonist can steal items or fight over resources. I quess this patch was to prevent that

Reply:
Being redone right now.

Reply:
Most removals are for a rework